sparts/tasks/dbus.py

```diff
@@ -24,6 +26,11 @@
 import glib
 import time
 
+# always init threads before calling any dbus code
+glib.threads_init()
```

Comment:
Is there a way to leave this in DBusMainLoopTask?  Dependency resolution should force its' initTask() to execute first before any other tasks' (who might depend on it?) 

Initializing stuff like this in module scope is problematic for a variety of reasons and I'd like to avoid it if possible.